client/blocks/eligibility-warnings/warning-list.tsx

```diff
@@ -77,14 +93,7 @@ function getWarningDescription(
 	switch ( context ) {
 		case 'plugin-details':
 		case 'plugins':
-			return translate(
-				'By installing a plugin the following change will be made to the site:',
-				'By installing a plugin the following changes will be made to the site:',
-				{
-					count: warningCount,
-					args: warningCount,
-				}
-			);
+			return '';
```

Comment:
We are not using this message for plugins anymore

Reply:
Do we need to limit this change to `plugin-details`? Not sure but just noticed plugins is a non modal context: https://github.com/Automattic/wp-calypso/pull/68365#issuecomment-1260517821